testing/acceptance_test.py

```diff
@@ -626,13 +627,8 @@ def test_cmdline_python_namespace_package(self, testdir, monkeypatch):
         #             ├── __init__.py
         #             └── test_world.py
 
-        def join_pythonpath(*dirs):
-            cur = os.environ.get("PYTHONPATH")
-            if cur:
-                dirs += (cur,)
-            return os.pathsep.join(str(p) for p in dirs)
-
-        monkeypatch.setenv("PYTHONPATH", join_pythonpath(*search_path))
+        # NOTE: the different/reversed ordering is intentional here.
+        monkeypatch.setenv("PYTHONPATH", prepend_pythonpath(*search_path))
         for p in search_path:
             monkeypatch.syspath_prepend(p)
```

Comment:
down below there is a `os.chdir` might br worth to use monkeypatch there as well